src/consumed.py

```diff
@@ -277,7 +282,7 @@ def output_results(self):
         for ba in self.regions:
             if (ba in self.import_regions) or (ba in self.generation_regions):
```

Comment:
@gailin-p what is happening with this line of code? What do we want to have happen if the ba is an import region or generation region? Should the next if statement be an `elif`?

Reply:
This just skips the rest of the loop for regions that are generation only (self.generation_regions) or regions that are outside the US but trade with US regions (self.import_regions): ie regions where we don't want to export files